### DISCUSSION TITLE
Can stories be sorted alphabetically?

### DISCUSSION THREAD

**Discussion by bpringe** [storybookjs/react-native#333] [Q&A] [answered]
I know this can be done in React with Storybook, as [documented here](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories), but can the same be done with React Native?

I tried to do so as documented for React, but it seems that does not work, or perhaps I did something incorrectly.

**Comment by dannyhw** [comment, marked as answer]
I believe this isn't currently possible, although thats definitely something that we should add.

**Reply by bpringe** [reply]
Thanks, that would be a great feature. As a workaround, I suppose I could write a script to sort the imports in the index file, and then run it as we add new stories. It's not ideal, of course. 😄

**Reply by pke** [reply]
Isn't this package using storybook core as its base? I struggled with exactly this sorting issue now for some hours and just read the storybook docs, which claimed it should be possible ;)

**Reply by dannyhw** [reply]
@pke it may be possible with the changes in #384 since there is more direct use of the storybook core in this implementation though I haven't tried this specific feature.
